### PR TITLE
add portainer stack deployment

### PR DESCRIPTION
**Pull request by LGinC**

### CODE REVIEW

File: content/LGinC/drone-portainer/index.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
+---
+date: 2020-07-16T00:00:00+00:00
+title: "Portainer Stack Deploy"
+author: "LGinC"
+tags: [ deploy ]
+repo: "turinge/drone-portainer"
```
Comment (tboerger) [inline content/LGinC/drone-portainer/index.md:6]: The repo doesn't match the folder of this file, so which one is the right one now?

File: content/LGinC/drone-portainer/index.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+logo: "portainer.svg"
+image: "turinge/drone-portainer"
+---
+
+The drone plugin can be used to deploy a Docker image to a Drone environment.The below pipeline configuration demonstrates simple usage:
```
Comment (tboerger) [inline content/LGinC/drone-portainer/index.md:11]: Docker image to a Drone environment?

File: content/LGinC/drone-portainer/index.md [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+: portainer server url. like this: http://xxx.com:9000
+
+username
+: portainer username
+
+password
+: portainer password
+
+endpointId
+: portainer endpoint id,default 1, localhost is 1 <br> portainer终结点id，默认是1,即第一个，一般为localhost
```
Comment (tboerger) [inline content/LGinC/drone-portainer/index.md:46]: Please drop or translate these Chinese strings.

File: content/LGinC/drone-portainer/index.md [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+
+stackname
+: name of stack <br> 服务栈的名称，会在stacks列表里显示
```
Comment (tboerger) [inline content/LGinC/drone-portainer/index.md:49]: Please drop or translate these Chinese strings.

File: content/LGinC/drone-portainer/index.md [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+
+imagename
+: name of pull image, like: mcr.microsoft.com/dotnet/core/aspnet:3.1-alpine <br> 将会进行拉取镜像的镜像名
```
Comment (tboerger) [inline content/LGinC/drone-portainer/index.md:52]: Please drop or translate these Chinese strings.

File: content/LGinC/drone-portainer/index.md [hunk 6]
```diff
@@ -0,0 +1,64 @@
...
+
+docker_compose
+: content of docker-compose.yml. ps: portainer just support version: "2" <br> docker-compose.yml 的内容 注意:portainer目前仅支持version: "2" <br>
```
Comment (tboerger) [inline content/LGinC/drone-portainer/index.md:55]: Please drop or translate these Chinese strings.

File: content/LGinC/drone-portainer/index.md (hunk 1 above)
Comment (LGinC) [reply content/LGinC/drone-portainer/index.md:6]: I had added it into readme

File: content/LGinC/drone-portainer/index.md (hunk 2 above)
Comment (LGinC) [reply content/LGinC/drone-portainer/index.md:11]: content of Dockerfile

```
FROM alpine

ADD "entrypoint.sh" "/entrypoint.sh"
RUN sed -i "s@dl-cdn.alpinelinux.org@mirrors.aliyun.com@g" /etc/apk/repositories && \
 apk -Uuv --no-cache add curl jq bash ca-certificates && chmod +x entrypoint.sh
ENTRYPOINT ["/entrypoint.sh"]
```

File: content/LGinC/drone-portainer/index.md (hunk 3 above)
Comment (LGinC) [reply content/LGinC/drone-portainer/index.md:46]: first line is translation of Chinese strings

File: content/LGinC/drone-portainer/index.md (hunk 4 above)
Comment (LGinC) [reply content/LGinC/drone-portainer/index.md]: updated

File: content/LGinC/drone-portainer/index.md (hunk 6 above)
Comment (LGinC) [reply content/LGinC/drone-portainer/index.md:55]: first line is translation of Chinese strings

File: content/LGinC/drone-portainer/index.md (hunk 3 above)
Comment (tboerger) [reply content/LGinC/drone-portainer/index.md:46]: But why the chinese translations? There is no chinese version of the index.

Comment (LGinC) [reply content/LGinC/drone-portainer/index.md:46]: can I create a Chinese_index.md? and add link in index.md

File: content/LGinC/drone-portainer/index.md (hunk 1 above)
Comment (tboerger) [reply content/LGinC/drone-portainer/index.md:6]: But the path to this markdown document still doesn't match the repo value.

File: content/LGinC/drone-portainer/index.md (hunk 2 above)
Comment (tboerger) [reply content/LGinC/drone-portainer/index.md:11]: Deploy a docker image to drone environment doesn't make sense, I don't get what you want to say with that.

File: content/LGinC/drone-portainer/index.md (hunk 3 above)
Comment (tboerger) [reply content/LGinC/drone-portainer/index.md:46]: The plugin index is English only, there is no Chinese version and it doesn't make sense to start that as none of the core maintainers is able to verify any translation beside the effort to maintain that at all.

File: content/LGinC/drone-portainer/index.md (hunk 1 above)
Comment (LGinC) [reply content/LGinC/drone-portainer/index.md]: has updated path match to repo

File: content/LGinC/drone-portainer/index.md (hunk 2 above)
Comment (LGinC) [reply content/LGinC/drone-portainer/index.md]: deploy docker-compose to portainer site

```
docker_compose: |
  version: "2"
  myapi:
    image: xxx/xx
```

this operation will generate a stack in portainer

File: content/LGinC/drone-portainer/index.md (hunk 3 above)
Comment (LGinC) [reply content/LGinC/drone-portainer/index.md]: dropped it

File: content/LGinC/drone-portainer/index.md (hunk 2 above)
Comment (LGinC) [reply content/LGinC/drone-portainer/index.md]: [https://app.swaggerhub.com/apis/deviantony/Portainer/1.23.2/#/stacks/StackCreate](https://app.swaggerhub.com/apis/deviantony/Portainer/1.23.2/#/stacks/StackCreate)

File: content/LGinC/drone-portainer/index.md (hunk 1 above)
Comment (tboerger) [reply content/LGinC/drone-portainer/index.md]: But https://github.com/turinge/drone-portainer results in a 404, so you added a wrong reference. repo is the name of the Git repository, image ist the name of the Docker image.

File: content/LGinC/drone-portainer/index.md (hunk 2 above)
Comment (tboerger) [reply content/LGinC/drone-portainer/index.md]: Than it should be still something like thsi:

```
This plugin can be used to deploy a Docker image to Portainer. The below pipeline configuration demonstrates simple usage:
```

File: content/turinge/drone-portainer/index.md [hunk 7]
```diff
@@ -0,0 +1,64 @@
...
+date: 2020-07-16T00:00:00+00:00
+title: Portainer Stack Deploy
+author: turinge
+tags: [ deployment]
+logo: portainer.svg
+repo: turinge/drone-portainer
+image: turinge/drone-portainer
+---
+
+The drone plugin can be used to deploy a Docker image to a Drone environment. [source](https://github.com/LGinC/drone-portainer) <br>
```
Comment (tboerger) [inline content/turinge/drone-portainer/index.md:11]: Please remove the `<br>`.

File: content/turinge/drone-portainer/index.md [hunk 8]
```diff
@@ -0,0 +1,64 @@
...
+: portainer server url. like this: http://xxx.com:9000
+
+username
+: portainer username
+
+password
+: portainer password
+
+endpointId
+: portainer endpoint id,default 1, localhost is 1
```
Comment (tboerger) [inline content/turinge/drone-portainer/index.md:47]: `portainer endpoint id, default 1, localhost is 1` and add an empty line after that.

File: content/turinge/drone-portainer/index.md [hunk 9]
```diff
@@ -0,0 +1,64 @@
...
+stackname
+: name of stack, show in stack list
+
+imagename
+: name of pull image, like: mcr.microsoft.com/dotnet/core/aspnet:3.1-alpine
+
+docker_compose
+: content of docker-compose.yml. ps: portainer just support version: "2"  <br>
```
Comment (tboerger) [inline content/turinge/drone-portainer/index.md:55]: `content of docker-compose, portainer supports version 2 only` should be fine, remove the rest.

File: content/LGinC/drone-portainer/index.md (hunk 1 above)
Comment (LGinC) [reply content/LGinC/drone-portainer/index.md]: update it.
[lginc/drone-portainer](https://hub.docker.com/r/lginc/drone-portainer)
[https://github.com/LGinC/drone-portainer](https://github.com/LGinC/drone-portainer)

File: content/LGinC/drone-portainer/index.md (hunk 2 above)
Comment (LGinC) [reply content/LGinC/drone-portainer/index.md]: done

File: content/turinge/drone-portainer/index.md (hunk 7 above)
Comment (LGinC) [reply content/turinge/drone-portainer/index.md]: done

File: content/turinge/drone-portainer/index.md (hunk 8 above)
Comment (LGinC) [reply content/turinge/drone-portainer/index.md]: done

File: content/turinge/drone-portainer/index.md (hunk 9 above)
Comment (LGinC) [reply content/turinge/drone-portainer/index.md]: done

**Comment by d1wilko** [comment]
Hi,

We have recently updated the drone plugin index and docs are no longer written in markdown - they are now a more consistent yaml format e.g. [docker](https://github.com/drone/drone-plugin-index/blob/master/plugins/docker/content.yaml)

Please have a look at the new readme if you would like to run the new plugin index [readme](https://github.com/drone/drone-plugin-index/blob/master/README.md)

If you would still like to add documentation to the plugins index can you change the format to
- An individual folder per plugin inside the `plugins` folder
- a `content.yaml` file that follows the format in the example above - this is for quick discovery, not full documentation
- an `original.md` file that can contain the markdown you have already written
- a logo image inside the `public/logos` folder

Thanks :)

**Pull request closed without merging**